spec/lib/phony/countries_spec.rb

```diff
@@ -797,7 +797,7 @@ def self.it_splits number, expected
       it_splits '299691123', ['299', '691', '123']
     end
     describe 'Guadeloupe (French Department of)' do
-      it_splits '590123456789', %w(590 123 456 789)
+      it_splits '590590456789', %w(590 590 45 67 89)
```

Comment:
I am a bit confused about the country code / national code duplication here. Is there an example where the CC is not equal to the NDC?